packages/commuter-client/src/reducers/index.js

```diff
@@ -6,6 +6,10 @@ const commuter = (state = {}, action) => {
     case types.REQUEST_CONTENTS:
       return Object.assign({}, state, { isFetching: action.isFetching });
     case types.RECEIVE_CONTENTS:
+    // filter out hidden files
+    action.entry.content = action.entry.content.filter((file) => {
+      return !file.name.startsWith('.')
```

Comment:
You'll need a conditional on the `entry.type` since this only applies to `directory`.